# perl shebang in scripts/dpkg-* is replaced by $(PERL) at build time,
# but $(PERL) is the path to native perl in sysroots. $(PERL) is also used
# by Makefile at build time, so cannot replace it by target path.
# This patch just removes the shebang conversion from Makefile.

diff --git a/scripts/Makefile.am b/scripts/Makefile.am
index fcd6006..7ca8efe 100644
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -109,8 +109,7 @@ nobase_dist_perllib_DATA = \
 man3_MANS =
 
 do_perl_subst = $(AM_V_GEN) \
-		sed -e "s:^\#![:space:]*/usr/bin/perl:\#!$(PERL):" \
-		    -e "s:\$$dpkglibdir[[:space:]]*=[[:space:]]*['\"][^'\"]*['\"]:\$$dpkglibdir=\"$(pkglibdir)\":" \
+		sed -e "s:\$$dpkglibdir[[:space:]]*=[[:space:]]*['\"][^'\"]*['\"]:\$$dpkglibdir=\"$(pkglibdir)\":" \
 		    -e "s:\$$pkgdatadir[[:space:]]*=[[:space:]]*['\"][^'\"]*['\"]:\$$pkgdatadir=\"$(pkgdatadir)\":" \
 		    -e "s:\$$admindir[[:space:]]*=[[:space:]]*['\"][^'\"]*['\"]:\$$admindir=\"$(admindir)\":" \
 		    -e "s:\$$version[[:space:]]*=[[:space:]]*['\"][^'\"]*[\"']:\$$version=\"$(PACKAGE_VERSION)\":"
